.github/workflows/github-actions-ci.yml

```diff
@@ -22,30 +21,34 @@ jobs:
           echo "List of changed files:"
           echo ${CHANGED_PYFILES}
           flake8 ${CHANGED_PYFILES} --count
-
+  
   ecephys_etl_pipelines:
-    name: ${{ matrix.python-version }}, ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: ["macos-latest", "windows-latest", "ubuntu-latest"]
-        python-version: ["3.6", "3.7"]
-    defaults:
-      run:
-        shell: bash -l {0}
+    runs-on: "ubuntu-latest"
     steps:
-      - uses: actions/checkout@v2
-      - uses: conda-incubator/setup-miniconda@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
         with:
-          auto-update-conda: true
-          python-version: ${{ matrix.python-version }}
-          activate-environment: test-env
-      - name: Install
+          username: alleninstitutepika
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build Docker Image
         run: |
-          conda activate test-env
-          pip install codecov
-          pip install -r test_requirements.txt
-          pip install .
-      - name: Test
+          git clone https://github.com/AllenInstitute/ecephys_etl_pipelines.git
+          cd ecephys_etl_pipelines
+          echo "Using branch $GITHUB_HEAD_REF"
+          git checkout $GITHUB_HEAD_REF
+          docker build --build-arg ECEPHYS_ETL_TAG=${GITHUB_HEAD_REF} --build-arg ECEPHYS_ETL_COMMIT_SHA=${GITHUB_SHA} --tag alleninstitutepika/ecephys_etl_pipelines:$GITHUB_SHA docker/
+      - name: Run Tests in Docker Image
         run: |
-          py.test --cov=ecephys_etl_pipelines -n 4
+          docker run alleninstitutepika/ecephys_etl_pipelines:$GITHUB_SHA pytest .
```

Comment:
Could you add something to `requirements.txt`, perhaps `scipy`, `numpy` or `h5py` and then make your dummy test try to use whichever package you choose?
I am suspicious that you are actually running these tests inside the conda environment, because it looks different to me.

Reply:
Oh thanks for catching that. Yeah just lookinf at it I could tell I eas never activating the environment, so I added that in. With a `pandas` in `requirements.txt` it builds and passes the tests now

Reply:
Oh thanks for catching it. Yeah just by looking at it I could see that I never activated the environment. That is taken care of now, and it's able to import `numpy`.